### PR TITLE
Add readme chinese translation

### PR DESCRIPTION
**Pull request by Promacanthus**
Add readme chinese translation.

https://github.com/golang-standards/project-layout/issues/66

### CODE REVIEW

File: README_zh-CN.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+### `/internal`
+
+私有的应用程序代码库。这些是不希望被其他人导入的代码。请注意：这种模式是Go编译器强制执行的。详细内容情况Go 1.4的[release notes](https://golang.org/doc/go1.4#internalpackages)。再次注意，在项目的目录树中的任意位置都可以有`internal`目录，而不仅仅是在顶级目录中。
+
+可以在内部代码包中添加一些额外的结构，来分隔共享和非共享的内部代码。这不是必选项（尤其是在小项目中），但是有一个直观的包用途是很棒的。应用程序实际的代码可以放在`/internal/app`目录（如，`internal/app/myapp`），而应用程序的共享代码放在`/internal/pkg`目录（如，`internal/pkg/myprivlib`）中。
+
+### `/pkg`
+
+这里存放外部应用策划给你需可以是使用的库代码（如，`/pkg/mypubliclib`）。其他项目将会导入这些库来保证项目可以正常运行，所以在将代码放在这里前，一定要三四而行。请注意，`internal`目录是一个更好的选择来确保项目私有代码不会被其他人导入，因为这是Go强制执行的。使用`/pkg`目录来明确表示代码可以被其他人安全的导入仍然是一个好方式。Travis Jeffery撰写的关于 [I’ll take pkg over internal](https://travisjeffery.com/b/2019/11/i-ll-take-pkg-over-internal/) 文章很好地概述了`pkg`和`inernal`目录以及何时使用它们。
```
Comment (zhaoying818) [inline README_zh-CN.md:54]: `这里存放外部应用策划给你需可以是使用的库代码` 不太对，应该是 `这里存放外部应用策划给你需可以是使用的库代码`

File: README_zh-CN.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+当根目录中包含很多非Go组件和目录时，将Go代码组织在一个地方有助于更人欧冠难以的运行多种不同的Go工具（在如下的文章中都有提到：2018年GopherCon [Best Practices for Industrial Programming](https://www.youtube.com/watch?v=PTE4VJIdHPg)，[Kat Zien - How Do You Structure Your Go Apps](https://www.youtube.com/watch?v=oL6JBUk6tj0) ，Golab 2018 [Massimiliano Pippi - Project layout patterns in Go](https://www.youtube.com/watch?v=3gQa1LWwuzk)）。
```
Comment (zhaoying818) [inline README_zh-CN.md:56]:
```suggestion
当您的根目录包含大量非Go组件和目录时，这也是一种将Go代码分组到一个位置的方法，从而使运行各种Go工具更加容易（在如下的文章中都有提到：2018年GopherCon [Best Practices for Industrial Programming](https://www.youtube.com/watch?v=PTE4VJIdHPg)，[Kat Zien - How Do You Structure Your Go Apps](https://www.youtube.com/watch?v=oL6JBUk6tj0) ，Golab 2018 [Massimiliano Pippi - Project layout patterns in Go](https://www.youtube.com/watch?v=3gQa1LWwuzk)）。
```

**Comment by kcq** [comment]
@Promacanthus do you mind taking a look at the conflicts... I'm not an expert here :) Thanks in advance!

File: README_zh-CN.md (hunk 1 above)
Comment (opso-code) [reply README_zh-CN.md]: there should be  `一定要三思而行` instead of `一定要三四而行`

**Comment by Promacanthus** [comment]
Fix file conflict

**Comment by kcq** [comment]
thanks for the PR! do you mind double checking once the PR is merged?

**Pull request merged**